src/test/java/com/zufar/icedlatte/favorite/api/endpoint/FavoriteListEndpointTest.java

```diff
@@ -0,0 +1,140 @@
+package com.zufar.icedlatte.favorite.api.endpoint;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.github.fge.jackson.JsonLoader;
+import com.zufar.icedlatte.favorite.endpoint.FavoritesEndpoint;
+import com.zufar.icedlatte.security.endpoint.UserSecurityEndpoint;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.boot.testcontainers.service.connection.ServiceConnection;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.io.IOException;
+
+import static com.zufar.icedlatte.test.config.RestAssertion.assertRestApiEmptyBodyResponse;
+import static com.zufar.icedlatte.test.config.RestAssertion.assertRestApiOkResponse;
+import static io.restassured.RestAssured.given;
+
+@Testcontainers
+@DisplayName("FavoriteListEndpointTest Tests")
+@ActiveProfiles("test")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class FavoriteListEndpointTest {
+
+    @Container
+    @ServiceConnection
+    static PostgreSQLContainer<?> postgres = new PostgreSQLContainer<>("postgres:13.11-bullseye");
+
+    @LocalServerPort
+    protected Integer port;
+
+    @Value("${jwt.secret}")
+    protected String secretKey;
+
+    @Value("${jwt.expiration}")
+    protected Long expiration;
+
+    @Value("${jwt.email}")
+    protected String email;
+
+    @Value("${jwt.password}")
+    protected String password;
+    protected static String jwtToken = "";
```

Comment:
consider to use this variable locally and return it from getJwtToken method and then reuse it in tokenAndSpecification()